AzureSpatialAnchorsSample/Assets/Scripts/SpatialAnchorsSample.cs

```diff
@@ -77,7 +77,7 @@ private async void Start()
             m_spatialAnchorManager.AnchorLocated += SpatialAnchorManagerAnchorLocated;
             m_spatialAnchorManager.LocateAnchorsCompleted += (sender, args) => Debug.Log("Locate anchors completed!");
 
-            await CreateSession();
```

Comment:
CreateSession may fail in Start() since it requires the spatialAnchorManager to have run its Start() previously.

Reply:
Is this something where `spatialAnchorManager` can do the needed work in `Awake()` instead?
Though looking at it, I guess that's outside our package and sample, so maybe something we can follow up on in the future.

Reply:
Consider to put some comments to the code instead of PR.  and put "Debug.LogError" on error conditions.